.gitattributes

```diff
@@ -1,4 +1,4 @@
 docs/*  linguist-documentation
 *.s linguist-generated
 lib/handlers/asm-docs.js linguist-generated
-test/cases/* linguist-generated
+test/* linguist-generated
```

Comment:
This of course drops our legitimate javascript tests in our counts, which seems a little unfair.

Reply:
I've changed that to `test/*-cases/*`.

Reply:
Thanks! and thanks again for taking the time to improve Compiler Explorer!